templates/question_rate.mustache

```diff
@@ -140,7 +140,7 @@
         {{#coltext}}{{{.}}}{{/coltext}}
         {{#colhiddentext}}<span class="accesshide">{{{.}}}</span>{{/colhiddentext}}
         {{#colinput}}<input type="radio" name="{{name}}" id="{{id}}" value="{{value}}"{{#checked}} checked="checked"{{/checked}}{{#disabled}} disabled="disabled"{{/disabled}}{{#onclick}} onclick="{{.}}"{{/onclick}}>
-        {{#label}}<label for="{{id}}" class="accesshide">{{{.}}}</label>{{/label}}{{/colinput}}
+        {{#label}}<label for="{{id}}" class="accesshide" aria-label="{{{.}}}">{{{.}}}</label>{{/label}}{{/colinput}}
```

Comment:
We need to add meaningful names to aria-labels. Do we know what the output of this "." will be @bryce-dev101 ?

Reply:
It will give the same value as the value of the label, I think when this was first done, it was done in an ambiguous way